Inventory.cs

```diff
@@ -0,0 +1,82 @@
+﻿public class Inventory
+{
+    private List<Product> products = new();
+
+    public void Add(string name, decimal price, int quantity)
+    {
+        var newProduct = new Product(name, price, quantity);
+        if (Product.AreValid(price, quantity)) 
+        {
+            products.Add(newProduct);
+        }
+    }
+
+    public bool IsEmpty()
+    {
+        return products.Count == 0;
+    }
+
+    public void Print(int index = -1)
```

Comment:
If I read this function, I wouldn't understand that when `index = -1` i should be printing all products. I  would just have two print functions, `PrintProduct(Index)` and `PrintAllProducts()`